src/modules/mc_pos_control/MulticopterPositionControl.cpp

```diff
@@ -88,6 +100,35 @@ int MulticopterPositionControl::parameters_update(bool force)
 		ModuleParams::updateParams();
 		SuperBlock::updateParams();
 
+		if (_param_sys_vehicle_resp.get() >= 0.f) {
+			// make it less sensitive at the lower end
+			float responsiveness = _param_sys_vehicle_resp.get() * _param_sys_vehicle_resp.get();
```

Comment:
That means the recommended responsiveness range gives a value between 0 and 4. I would constrain it to 0.05 - 1 for easier sacling.

Reply:
Right, that is intended. I limited the max to 1 for now, but I wanted to make sure we can go beyond that (you know how I like my vehicles :)